taworks/taform/views.py

```diff
@@ -235,7 +244,21 @@ def apply(request):
                     }
                 return render(request, 'taform/application.html', context)
             context = None
-            return HttpResponseRedirect('application_submitted.html')
+            print app_id
+            courses_applied= models.Course.objects.filter(application__student_id=app_id, application__preference__in = [1,2,3]).distinct()
```

Comment:
this filter doesn't look right.

Reply:
explain?

Reply:
models.Course.objects.filter(application__student_id=app_id, application__preference__in = [1,2,3]).distinct()

I think it should be student__id